pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator_test.go

```diff
@@ -414,42 +414,46 @@ var _ = Describe("VirtualMachineInstance Mutator", func() {
 		),
 	)
 
-	DescribeTable("should add the default network interface",
-		func(iface string) {
-			expectedIface := "bridge"
-			switch iface {
-			case "masquerade":
-				expectedIface = "masquerade"
-			case "slirp":
-				expectedIface = "slirp"
-			}
+	Context("should add network interfaces", func() {
+		var kvConfig *v1.KubeVirt
 
-			permit := true
-			testutils.UpdateFakeKubeVirtClusterConfig(kvInformer, &v1.KubeVirt{
+		BeforeEach(func() {
+			kvConfig = &v1.KubeVirt{
 				Spec: v1.KubeVirtSpec{
 					Configuration: v1.KubeVirtConfiguration{
-						NetworkConfiguration: &v1.NetworkConfiguration{
-							NetworkInterface:     expectedIface,
-							PermitSlirpInterface: &permit,
-						},
+						NetworkConfiguration: &v1.NetworkConfiguration{},
 					},
 				},
-			})
+			}
+		})
+
+		It("should add the bridge network interface", func() {
```

Comment:
Perhaps replace `add` with `use`. Same for the others.

Reply:
Done